azkaban-common/src/main/java/azkaban/executor/container/KubernetesContainerizedImpl.java

```diff
@@ -1361,8 +1368,8 @@ private void addInitContainers(final int executionId,
     if (this.prefetchAllCredentials) {
       try {
         final String imageFullPath =
-            versionSet.getVersion("security-init").get().pathWithVersion();
-        v1SpecBuilder.addInitContainerType(imageFullPath, ImagePullPolicy.IF_NOT_PRESENT,
+            versionSet.getVersion(InitContainerType.SECURITY.initPrefix).get().pathWithVersion();
```

Comment:
It seems it should be better renamed to this.azkabanSecurityInitImageName? It should be the image name.